README.md

```diff
@@ -42,13 +42,13 @@ var results = linter.verify({ source: template, moduleId: 'template.hbs' });
 
 `results` will be an array of objects which have the following properties:
 
-* `rule` - The name of the rule that triggered this warning/error.
-* `message` - The message that should be output.
-* `line` - The line on which the error occurred.
-* `column` - The column on which the error occurred.
-* `moduleId` - The module path for the file containing the error.
-* `source` - The source that caused the error.
-* `fix` - An object describing how to fix the error.
+- `rule` - The name of the rule that triggered this warning/error.
+- `message` - The message that should be output.
+- `line` - The line on which the error occurred.
+- `column` - The column on which the error occurred.
+- `moduleId` - The module path for the file containing the error.
+- `source` - The source that caused the error.
+- `fix` - An object describing how to fix the error.
```

Comment:
Seems like you ran prettier on this file, can you revert the changes that are unrelated to your PR here?

I'd love to have a separate PR to make this file formatted how prettier wants it to be, but I want to keep stylistic changes separate from those that introduce functionality...